app/models/liabilitycalculation/viewmodels/TaxYearSummaryViewModel.scala

```diff
@@ -44,9 +43,35 @@ object TaxYearSummaryViewModel {
       deductions = calc.calculation.flatMap(c => c.taxCalculation.map(tc => tc.incomeTax.totalAllowancesDeductionsReliefs)).getOrElse(0.00),
       totalTaxableIncome = calc.calculation.flatMap(c => c.taxCalculation.map(_.incomeTax.totalTaxableIncome)).getOrElse(0),
       forecastIncome = calc.calculation.flatMap(c => c.endOfYearEstimate.flatMap(_.totalEstimatedIncome)),
-      forecastIncomeTaxAndNics = calc.calculation.flatMap(c => c.endOfYearEstimate.flatMap(_.incomeTaxNicAndCgtAmount))
+      forecastIncomeTaxAndNics = calc.calculation.flatMap(c => c.endOfYearEstimate.flatMap(_.incomeTaxNicAndCgtAmount)),
+      inYearCalculationRunDate = calc.metadata.calculationTimestamp.flatMap(timestamp => getInYearCalculationRunDate(timestamp, taxYear))
     )
   }
+
+  def isUnattendedCalc(calculationReason: Option[String]): Boolean = calculationReason match {
+    case Some("unattendedCalculation") => true
+    case _ => false
+  }
+
+  def getInYearCalculationRunDate(calculationTimestamp: String, taxYear: Int): Option[String] = {
+    val firstQuarterstart = LocalDate.parse(s"${taxYear - 1}-04-05")
```

Comment:
you might scratch surface for the next ticket => https://jira.tools.tax.service.gov.uk/browse/MISUV-3215
I would suggest to create singleton DateService and create lambdas for: firstQuarterEnd .. to fourthQuarterEnd, so these could be re-used + method to convert calculationTimestamp into Either[Error,LocalDate];

Also I would think that calculationTimestamp method could be move under DateService singleton as well and you can call it were required; 
Maybe worth to model new data type. ie. enum for year quarters? not sure if we can trust/rely on strings